tests/plots/test_declarative.py

```diff
@@ -1254,10 +1255,10 @@ def test_declarative_upa_obs():
     return pc.figure
 
 
-@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0.11)
+@pytest.mark.mpl_image_compare(remove_text=True, tolerance=0)
 @needs_cartopy
 def test_declarative_upa_obs_convert_barb_units():
```

Comment:
Looks like this one consistently shifts a few pixels across OSes? Seems to be the only other test failing. Do you prefer to leave the boundaries in `test_declarative_upa_obs` as well?

Reply:
Bumped the tolerance. I kept the map since it's the only test that uses `'land'` and produces an image.